### PR TITLE
Fix the duplicate test name

### PR DESCRIPTION
**Pull request by pankajastro**
The test with def `test_get_dag_params` is twice declare in test_dagbuilder.py. This PR update the test name.

### CODE REVIEW

File: tests/test_dagbuilder.py [hunk 1]
```diff
@@ -37,7 +36,7 @@
 from dagfactory import dagbuilder
 
 if version.parse(AIRFLOW_VERSION) >= version.parse("2.0.0"):
-    from airflow.timetables.interval import CronDataIntervalTimetable
+    pass
```
Comment (tatiana) [inline tests/test_dagbuilder.py:39]: Was this a dead code?

Comment (pankajastro) [reply tests/test_dagbuilder.py]: Removed it

File: tests/test_dagbuilder.py [hunk 2]
```diff
@@ -461,7 +454,7 @@ def test_build():
         assert actual["dag"].tags == ["tag1", "tag2"]
 
 
-def test_get_dag_params():
+def test_get_dag_params_default_args():
```
Comment (pankajkoti) [inline tests/test_dagbuilder.py:457]:
```suggestion
def test_get_dag_params_dag_with_task_group():
```

I am not sure `default_args` if a good suffix here. The other test `test_get_dag_params` also gets `default_args`. The difference in both the tests is that the other one passes the `DAG_CONFIG` and this one passes `DAG_CONFIG_TASK_GROUP` to DagBuilder.

**Review by pankajkoti — APPROVED** [review]
LGTM. One minor suggestion about the naming of the test

**Pull request merged**